app/uk/gov/hmrc/fileupload/frontendGlobal.scala

```diff
@@ -125,8 +126,10 @@ class ApplicationModule(context: Context) extends BuiltInComponentsFromContext(c
 
   // quarantine
   lazy val quarantineRepository = quarantine.Repository(db)
-  lazy val retrieveFile = quarantineRepository.retrieveFile _
-  lazy val getFileFromQuarantine = QuarantineService.getFileFromQuarantine(retrieveFile) _
+  //lazy val retrieveFileFromQuarantineBucket = s3Service.retrieveFileFromQuarantine _
+  //TODO discuss alternative thread pools
+  lazy val ec = ExecutionContext.fromExecutorService(Executors.newFixedThreadPool(25))
```

Comment:
We will need to make a decision on thread pools for all blocking calls. We will discuss a bit later